_data/versions.yml

```diff
@@ -1,4 +1,55 @@
 v2.6:
+- title: v2.6.3
+  note: |
+    28 November 2017
+    Calico v2.6.3 addresses several outstanding common vulnerabilities and exposures (CVE).
+    - A new node controller for Kubernetes deployments clears data associated with deleted nodes from the Calico datastore, preventing conflicts that can lead to crash loops. 
```

Comment:
Is this really all that's changed? I bet there is more if we run the release-note script (though maybe not!)

Reply:
Yeah, I ran the release-note script and this is basically what showed up.

Reply:
Oh, I also heard that it might have adjusted the confd log level?

Reply:
guess we skipped setting the milestone for some of those last minute fixes, which leaves `generate-release-notes.py` empty handed.

Reply:
The confd debug->info went in as part of a much larger commit, unfortunately: https://github.com/projectcalico/calico/pull/1395. @fasaxc and I discussed whether to break that out into a separate PR, but skipped past it. So we'll have to generate a note for this manually.